src/components/Map/__stories__/ApiKeyInput.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useCallback, useState, useEffect, ChangeEvent} from 'react';
+
+import {Button} from '../../';
+import {useMapApiKey} from '../../../../src';
+
+interface ApiKeyInputProps {
+    id: string;
+}
+
+export const ApiKeyInput = (props: ApiKeyInputProps) => {
+    const {id} = props;
+    const [apiKey, setApiKey] = useState('');
+    const [_key, setKey] = useMapApiKey();
+
+    const onChange = useCallback(
+        (event: ChangeEvent<HTMLInputElement>) => {
+            setApiKey(event.target.value);
+        },
+        [setApiKey],
+    );
+
+    const onButtonClick = useCallback(() => {
+        localStorage.setItem(id, apiKey);
+
+        setKey(apiKey);
+    }, [id, apiKey, setKey]);
+
+    useEffect(() => {
+        if (apiKey) return;
+
+        const keyFromLS = localStorage?.getItem(id);
+
+        if (keyFromLS) {
+            setApiKey(keyFromLS);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    return (
+        <div>
+            <label style={{display: 'block'}}>Api key for google maps</label>
+            <input
```

Comment:
can we use TextInput from ui-kit?
https://preview.gravity-ui.com/uikit/?path=/story/components-textinput--default